src/engine/Union.cpp

```diff
@@ -180,11 +188,20 @@ void Union::computeUnion(
       // This if clause is only here to avoid creating the call to insert when
       // it would not be possible to call the function due to not matching
       // columns.
+      AD_CHECK(LEFT_WIDTH == OUT_WIDTH);
```

Comment:
Is this of the kind "This should never happen?"

If yes, do we still need the if in the next line now that we have the AD_CHECK?

Reply:
YES, we do, because the next line is `if constexpr` (the direct copy/insert operations would not compile if the columns do not match).